### PR TITLE
Modified user test to contain password and active fields

### PR DESCRIPTION
**Pull request by MaraJade**
issue #62

### CODE REVIEW

File: tests/users.js [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
+        it('returns all users in the database', function(done) {
             request.get(base_url + 'users', function(err, res) {
                 var bodyAsString = String.fromCharCode.apply(null, res.body);
                 var expected_results = [
                     {
                         id: 1,
-                        username: 'deanj'
+                        username: 'deanj',
+                        active: true,
+                        password: 'pass'
```
Comment (tschuy) [inline tests/users.js]: These should be salted bcrypt values. I can show you how to generate a few.

**Comment by tschuy** [comment]
jscs is failing. Looks like your line is too long. You can disable individual jscs rules for a block with
```
//jscs:disable maximumLineLength
var long_name = 'this is a super long var that is longer than 80 characters, which is bad but if it's a long string that's maybe 90 it's better than splitting, in my opinion';
//jscs:enable maximumLineLength

**Comment by MaraJade** [comment]
Should be implemented correctly now.

**Comment by tschuy** [comment]
Looks good to me.

**Comment by Kennric** [comment]
+1

**Comment by tschuy** [comment]
+1

**Pull request merged**